syntaxes/vscode-scl.tmLanguage.json

```diff
@@ -281,8 +300,8 @@
 					"captures": {
 						"2": { "name": "punctuation.definition.comment.scl"}
 					},
-					"begin": "\\(\/\\*",
-					"end": "\\*\/\\)",
+					"begin": "\\(/\\*",
```

Comment:
After some testing it sems these do the same thing. The interpreter is probably saving us. Should this instead be `\\(\\/\\*`  and `\\*\\/\\)`? Or do you have test that differentiate these? I assume the original here i.e. `\\(\/\\*` and `\\*\/\\)` is wrong even though it seems to catch `(/* some multiline multilang(?) comment */)`

Reply:
I have no idea why this worked and I don't remember why it was changed. I updated it to work with regex101 now.